### PR TITLE
SWS: Support python2, also support dr_trigger function

### PR DESCRIPTION
**Pull request by hamdanigbaria**
Changed a little bit in the code to support python2, also added support for dr_trigger function to be called as a stand alone function.
Example:
	python dr_trigger.py <DPDK_PID> <DPDK_PORT> <dump_file>

Signed-off-by: Hamdan Igbaria <hamdani@nvidia.com>

### CODE REVIEW

File: src/dr_trigger.py [hunk 1]
```diff
@@ -141,3 +141,19 @@ def trigger_dump(s_pid, s_port, path, s_flow_ptr):
     dump_file.close()
     sock.close()
     return path
+
+def main():
```
Comment (alexvesker) [inline src/dr_trigger.py:145]: is this change needed for dr_ste_v0?

File: src/parsers/dr_ste_v1_parser.py [hunk 2]
```diff
@@ -102,11 +103,10 @@ def mlx5_ifc_ste_v1_match_bits_parser(bin_str, definer_id, raw):
     ret["hash_after_actions"] = hex(int(bin_str[157: 158], 2))
     ret["reserved_at_9e"] = hex(int(bin_str[158: 160], 2))
 
-    ret["action0"] = hex(int(bin_str[160: 192], 2))
-
-    ret["action1"] = hex(int(bin_str[192: 224], 2))
-
-    ret["action2"] = hex(int(bin_str[224: 256], 2))
+    #Use the % for printin instead of hex() func here to not get the trailing L for Long numbers
+    ret["action0"] = "0x%x" % int(bin_str[160: 192], 2)
```
Comment (alexvesker) [inline src/parsers/dr_ste_v1_parser.py:107]: add new func: hex_str

File: src/dr_trigger.py (hunk 1 above)
Comment (hamdanigbaria) [reply src/dr_trigger.py:145]: no in dr_ste_v0 a different mechanism was used there

**Review by kliteyn — APPROVED** [review]
